CONTRIBUTING.md

```diff
@@ -97,24 +163,23 @@ As new versions of Xcode and iOS are released, we have to upgrade both to stay u
 
 1. Change the value of `runs-on` in [`ci.yml`](./.github/workflows/ci.yml#26). The new value should be on of the [available environments](https://github.com/actions/virtual-environments/tree/main/images/macos) in GitHub Actions.
 1. Update the `BUILD_SDK` variable in [`Rakefile`](./Rakefile#5) to the new build SDK we should use.
-1. Update `correctMajorVersion` and `correctMinorVersion` in [`BPKSnapshotTest.`](./Example/SnapshotTests/BPKSnapshotTest.h).
+1. Update `correctMajorVersion` and `correctMinorVersion` in [`BPKSnapshotTest`](./Example/SnapshotTests/BPKSnapshotTest.h).
 1. Update `expectedMajorVersion` and `expectedMinorVersion` in [`BPKSnapshotTest.swift`](./Example/SnapshotTests/BPKSnapshotTest.swift#26).
 1. Run all snapshot tests.
 1. **Review the failing snapshots thoroughly.** Most likely, all snapshots will have changed, **but** the diffs should be miniscule and mostly to do with changes in Apple's fonts.
 1. **Run all snapshot tests in record mode.** At the time of writing this involves manually setting `recordMode` in every test case, we should have a better method than this, but alas we don't :(
 1. Manually test the example app with the new version.
 
-## Releasing
+</details>
+
+<details>
+    <summary>Releasing</summary>
 
 > Backpack team only
 
 To issue a new release:
-
 * Publish draft release
+</details>
```

Comment:
Given the number of lines this actually covers does it need to be hidden in an expandable?